proto/cometbft/types/v1/params.proto

```diff
@@ -21,55 +21,62 @@ message ConsensusParams {
   FeatureParams   feature   = 7;
 }
 
-// BlockParams contains limits on the block size.
+// BlockParams define limits on the block size and gas.
 message BlockParams {
-  // Max block size, in bytes.
-  // Note: must be greater than 0
+  // Maximum size of a block, in bytes.
+  //
+  // Must be greater or equal to -1. If set to -1, the limit is the maximum
+  // size for a block, hard-coded to 100MB.
   int64 max_bytes = 1;
-  // Max gas per block.
-  // Note: must be greater or equal to -1
+  // Maximum gas wanted by transactions included in a block.
+  //
+  // Must be greater or equal to -1. If set to -1, no limit is enforced.
   int64 max_gas = 2;
 
   reserved 3;  // was TimeIotaMs see https://github.com/tendermint/tendermint/pull/5792
 }
 
-// EvidenceParams determine how we handle evidence of malfeasance.
+// EvidenceParams determine the validity of evidences of Byzantine behavior.
 message EvidenceParams {
-  // Max age of evidence, in blocks.
+  // Maximum age of evidence, in blocks.
   //
-  // The basic formula for calculating this is: MaxAgeDuration / {average block
-  // time}.
+  // The recommended formula for calculating it is max_age_duration / {average
+  // block time}.
   int64 max_age_num_blocks = 1;
 
-  // Max age of evidence, in time.
+  // Maximum age of evidence, in time.
   //
-  // It should correspond with an app's "unbonding period" or other similar
-  // mechanism for handling [Nothing-At-Stake
-  // attacks](https://github.com/ethereum/wiki/wiki/Proof-of-Stake-FAQ#what-is-the-nothing-at-stake-problem-and-how-can-it-be-fixed).
+  // The recommended value of is should correspond to the application's
+  // "unbonding period" or other similar mechanism for handling
+  // Nothing-At-Stake attacks.
+  // See: https://github.com/ethereum/wiki/wiki/Proof-of-Stake-FAQ#what-is-the-nothing-at-stake-problem-and-how-can-it-be-fixed.
   google.protobuf.Duration max_age_duration = 2
       [(gogoproto.nullable) = false, (gogoproto.stdduration) = true];
 
-  // This sets the maximum size of total evidence in bytes that can be committed in a single block.
-  // and should fall comfortably under the max block bytes.
-  // Default is 1048576 or 1MB
```

Comment:
We should not mention the default for `max_bytes` anymore?

Reply:
Probably yes, I'll look into.